src/ResourceManager/Websites/Commands.Websites/help/AzureRM.Websites.md

```diff
@@ -12,25 +12,25 @@ ARM (Azure Resource Manager) Web App and App Service Plan commands.
 
 ## AzureRM.Websites Cmdlets
 ### [Edit-AzureRmWebAppBackupConfiguration](Edit-AzureRmWebAppBackupConfiguration.md)
-Modifies an Azure Web App backup schedule.
+
```

Comment:
I don't know what happened here, but it deleted a bunch of the cmdlet descriptions. I'll fix it.